Formula/dafny.rb

```diff
@@ -37,10 +29,15 @@ def install
 
     libexec.install Dir["Binaries/*"]
 
+    cd libexec do
+      system "curl", "-sSL", "-o", "file.zip", "https://github.com/Z3Prover/z3/releases/download/z3-4.8.4/z3-4.8.4.d6df51951f4c-x64-osx-10.14.1.zip"
+      system "unzip", "-q", "file.zip"
```

Comment:
This should be done using a `resource` block, not through curl. `resource` blocks check source integrity by verifying `sha256`.

Have a look at another formula for examples. `grep  'resource do'` in the core repo would work.

Reply:
Also, this looks like it downloads something that is pre-built, which Homebrew doesn't like, I think.

Reply:
Fixed.